htdocs/js/smr15.js

```diff
@@ -102,8 +102,57 @@ function showRaceInfo(select) {
 	var desc = document.getElementById('race_descr');
 	$(desc).children().addClass('hide');
 	$(".race_descr" + race_id, desc).removeClass('hide');
+	graph_by_plotly(race_id);
 }
 
+// Used by game_join.php 
+function graph_by_plotly(race_id) {
```

Comment:
I think this could use a more descriptive function name, e.g. `createRaceRadarChart` or similar (camel case for consistency).